.github/workflows/ci.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Build Types
         run: npm run build:types
       - name: Test Types
-        run: npm run test:types
+        run: npm run build:types
```

Comment:
This is a duplicate there is a job “build:types” already

Reply:
removed